lib/filter-stream.js

```diff
@@ -48,22 +47,19 @@ function FilterStream( blockMap, options ) {
   this.bytesWritten = 0
   /** @type {Number} Current offset in bytes */
   this.position = 0
+  /** @type {Array<Object>} Ranges */
+  this.ranges = this._prepareRanges()
+  /** @type {Object} Range being currently processed */
+  this.currentRange = this.ranges.shift()
 
-  /**
-   * Internal buffer queue for chunking to block size,
-   * if source does not emit appropriately sized chunks
-   * @type {Buffer[]}
-   * @private
-   */
-  this._buffers = []
+  this.highWaterMark = 64 * 1024
```

Comment:
Just out of curiosity: is there any particular reason behind this high water mark number? If so, should we explain it in a comment?